react-express-mongodb/frontend/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Create image based on the official Node image from dockerhub
+FROM node:13.13.0-stretch-slim
```

Comment:
Just use a `node:13.13.0-stretch` instead a slim one, because python is needed during the build.

```
gyp ERR! find Python
gyp ERR! find Python Python is not set from command line or npm configuration
gyp ERR! find Python Python is not set from environment variable PYTHON
gyp ERR! find Python checking if "python" can be used
gyp ERR! find Python - "python" is not in PATH or produced an error
gyp ERR! find Python checking if "python2" can be used
gyp ERR! find Python - "python2" is not in PATH or produced an error
gyp ERR! find Python checking if "python3" can be used
gyp ERR! find Python - "python3" is not in PATH or produced an error
gyp ERR! find Python
```

Reply:
Done